refgenie/asset_build_packages.py

```diff
@@ -72,6 +72,32 @@
             "cut -f 1,2 {asset_outfolder}/{genome}.fa.fai > {asset_outfolder}/{genome}.chrom.sizes",
         ]
     },
+    "fasta_txome_extract": {
+        DESC: "cDNA sequences in FASTA format exracted from the genome and annotation files, indexed FASTA (produced with samtools index) and chromosome sizes file",
+        ASSETS: {
+            "fasta_txome": "{genome}.fa",
```

Comment:
to make use of `refgenie` defaults system one seek key should match the asset name, so `fasta_txome_extract` instead of `fasta_txome`

```refgenie --genome G --asset fasta_txome_extract``` should  return the path to the FASTA file.

Reply:
or, switch the asset name to `fasta_txome`, right?

Reply:
only if we decided to remove the current `fasta_txome` recipe

Reply:
is there an easy way to include some logic in the cmd list? 
Ideally this txome cration would be included inside the the fasta_txome asset itself, if a fasta_txome is provided as input then it's just copied to the destination path, otherwise the gtf+genome building approach is used.

Reply:
@ieguinoa that's a great idea. I suppose we could just write that somehow into the shell commands with shell conditionals.

so it would be something like

```
CMD_LST: [
    "if [ -f {tx_fasta} ]; then cp {tx_fasta} {asset_outfolder}/{genome}.fa.gz; gzip -df {asset_outfolder}/{genome}.fa.gz; else gunzip -c {gtf} > {asset_outfolder}/{genome}.gtf; gffread -g {fasta} -w {asset_outfolder}/{genome}.fa {asset_outfolder}/{genome}.gtf; fi"
    "samtools faidx {asset_outfolder}/{genome}.fa",
    "cut -f 1,2 {asset_outfolder}/{genome}.fa.fai > {asset_outfolder}/{genome}.chrom.sizes",
]
```

ugly, but could get the job done for now. we'd just have to change to 2 optional inputs; either you provide a tx_fasta, or a fasta and gtf.

the recipe system isn't really powerful enough to do this well, though. in the long term, we will definitely need to accommodate this scenario with custom recipes.